src/EditorFeatures/CSharpTest/InitializeParameter/AddParameterCheckTests.cs

```diff
@@ -249,6 +298,28 @@ public async Task TestNotOnPartialMethodDefinition2()
             var code = @"
 using System;
 
+partial class C
+{
+    partial void M(string s)
+    {
+    }
+
+    partial void M([||]string s);
+}";
+            await new VerifyCS.Test
+            {
+                LanguageVersion = LanguageVersion.Preview,
```

Comment:
For the cases where the test is the same other than changing language version, consider using a xUnit theory instead of a fact to avoid all the extra duplication.